Resources/views/CRUD/edit_orm_many_association_script.html.twig

```diff
@@ -34,6 +34,10 @@ This code manage the many-to-[one|many] association field popup
         initialize_popup_{{ id }}();
 
         var target = jQuery(this);
+        
+        if (target.attr('href').length === 0 || target.attr('href') === '#') {
```

Comment:
What if it is `#some-id` ? Or `javascript()` ? Isn't there some better way to detect valid urls? Cc @ju1ius